profiler.rst

```diff
@@ -3,6 +3,8 @@ Profiler
 
 .. toctree::
     :maxdepth: 1
-    :glob:
 
-    profiler/*
+    profiler/data_collector
+    profiler/profiling_data
+    profiler/matchers
+    profiler/storage
```

Comment:
I changed the order - since we can control the order with this method, let's put the best ones first!